### ISSUE TITLE
Update readme for 2.7

### ISSUE THREAD

**Issue by lat9** [DivaVocals/zen_COWOA#23]
Not that you guys don't know that the readme still references COWOA 2.6!  This issue is more of a reminder that the readme update is still needed.

**Comment by DivaVocals** [comment]
Yep.. :smile:  didn't plan to put much effort into updating it until ready to submit to the plugins repo.
